test/rpc-integration/default.nix

```diff
@@ -16,7 +16,7 @@ let
           export ${f}
         '') buildFlags}
         export SERVER=${kore-rpc-booster}/bin/kore-rpc-booster
-        export CLIENT=${rpc-client}/bin/rpc-client
+        export CLIENT="${rpc-client}/bin/rpc-client -l debug"
```

Comment:
I ran with debug logging to analyse the M1 failure but it is flaky... passed on the 2nd run after truncating some output on the first run.
Hopefully this won't become a problem later if/when we merge this.